core/v2.go

```diff
@@ -767,17 +767,56 @@ func (core *Core) v2API() *route.Router {
 		for _, archive := range aa {
 			archives[archive.UUID.String()] = archive
 		}
+		// check to see if we're offseting task requests
+		paginationDate, err := strconv.ParseInt(r.Param("before", "0"), 10, 64)
 
-		tasks, err := core.DB.GetAllTasks(
-			&db.TaskFilter{
+		var filter db.TaskFilter
+		if paginationDate > 0 {
+			filter = db.TaskFilter{
```

Comment:
Was this one missed?  Should be able to unify the filter object creation and not need the if.